cmd/minikube/cmd/service.go

```diff
@@ -148,6 +148,14 @@ You may select another namespace by using 'minikube service {{.service}} -n <nam
 					out.String(fmt.Sprintf("%s\n", serviceURLs))
 				}
 			}
+			// check whether all pods of this service is crashed
+			clientset, err := kapi.Client(cname)
+			if err != nil {
+				exit.Error(reason.InternalKubernetesClient, "Failed to acquire Kubernetes client", err)
+			}
```

Comment:
If you look at the other functions in `pkg/minikube/service/service.go` they all create the client within the file using `client, err := K8s.GetCoreClient(cname)` so I think it makes sense to reuse the func in that file and pass `cname` into `service.CheckServicePods`.

Reply:
Changed